tests/php/Domain/Services/CreateAccount.php

```diff
@@ -36,9 +33,8 @@ private function get_test_instance() {
 	private function execute_create_customer_from_order( $email, $first_name, $last_name, $options = [] ) {
 		/// -- Test-specific setup start.
 
-		// Can't log out the user in a unit test ("headers already sent" error).
-		// So - we are assuming the tests run in an environment where the user is logged out.
-		// wp_logout();
+		// Simulate (force) logged out user – if the user is logged in, no account will be created.
+		wp_set_current_user( 0 );
```

Comment:
Apologies, but it looks like there's no need to do this because the [WordPress test suite handles this in `tearDown`](https://github.com/WordPress/wordpress-develop/blob/19513ff6a0fc27b2ff7e9debeb6e94b7e1b4d658/tests/phpunit/includes/abstract-testcase.php#L169).  So effectively, the current user will already be not set for the test (which is why it worked fine without it).

We could probably remove.

Reply:
Right, and we can safely assume that the tests run with no user logged in (even if we were the first test, before any `tearDown`!). Removed: 81b3a0eea114e78379c59916f0cf13f1fc226590